src/main/groovy/wooga/gradle/github/base/GithubBasePlugin.groovy

```diff
@@ -20,39 +20,42 @@ package wooga.gradle.github.base
 import org.gradle.api.Action
 import org.gradle.api.Plugin
 import org.gradle.api.Project
-import org.gradle.api.Task
 import org.gradle.api.internal.ConventionMapping
-import org.gradle.api.specs.Spec
-import org.gradle.api.tasks.TaskContainer
+import wooga.gradle.github.base.internal.DefaultGithubPluginExtension
+import wooga.gradle.github.base.tasks.internal.AbstractGithubTask
 
+/**
+ * A base {@link org.gradle.api.Plugin} to register and set conventions for all {@link AbstractGithubTask} types.
+ *
+ */
 class GithubBasePlugin implements Plugin<Project> {
 
+    /**
+     * Value for github gradle convention.
+     * @value "github"
+     */
     static final String EXTENSION_NAME = "github"
-    static final String GROUP = "github"
 
-    private Project project
-    private TaskContainer tasks
+    /**
+     * Value for github gradle task group.
+     * @value "github"
+     */
+    static final String GROUP = "github"
 
     @Override
     void apply(Project project) {
-        this.project = project
-        this.tasks = project.tasks
+        def tasks = project.tasks
```

Comment:
is `tasks` used anywhere?

Reply:
Yes. I pass it to this sweet static method.